src/System.Management.Automation/engine/runtime/Operations/MiscOps.cs

```diff
@@ -1031,11 +1031,8 @@ internal FileRedirection(RedirectionStream from, bool appending, string file)
 
         public override string ToString()
         {
-            return string.Format(CultureInfo.InvariantCulture, "{0}> {1}",
-                                 FromStream == RedirectionStream.All
-                                     ? "*"
-                                     : ((int)FromStream).ToString(CultureInfo.InvariantCulture),
-                                 File);
+            return string.Format(CultureInfo.InvariantCulture, $"{0}> {File}",
+                                 FromStream == RedirectionStream.All ? "*" : ((int)FromStream).ToString(CultureInfo.InvariantCulture));
```

Comment:
The code can be insterted in {0}.
(In C# 11 even multilines are supported https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-11#newlines-in-string-interpolations but we should switch from C# 10 to 11 in our msbuild files.)